### DISCUSSION TITLE
How can I see dividend in paisa

### DISCUSSION THREAD

**Discussion by kiil** [ananthakumaran/paisa#190] [Q&A] [answered]
Some stock pay dividend, usually annually.

e.g

```
2023-05-01 Dividend SYVKUA
  Assets:Checking
  Income:Dividend:SYVKUA     $-25
```

Could this somehow be displayed on the `gain` page for Assets:Equity:SYVKUA?

**Comment by ananthakumaran** [comment, marked as answer]
Dividend is not supported yet.
